tests/test_types.py

```diff
@@ -0,0 +1,33 @@
+"""
+:Description: Unit tests for the types module
+"""
+
+from __future__ import annotations
+
+from conda_recipe_manager.types import MessageCategory, MessageTable
+
+
+def test_message_table_add_retrieve_and_clear() -> None:
+    message_table = MessageTable()
+    message_table.add_message(MessageCategory.ERROR, "Test error message")
+    message_table.add_message(MessageCategory.WARNING, "Test warning message")
+
+    assert message_table.get_totals_message() == "1 error and 1 warning were found."
+
+    assert message_table.get_message_count(MessageCategory.EXCEPTION) == 0
```

Comment:
I would prefer we have lots of little tests over one big test that checks many things. Small tests can be highly parallelized and narrow the search space of a potential problem.

I would also challenge you to see if you can parameterize these tests so we can build more tests faster.